README.md

```diff
@@ -115,147 +115,7 @@ make
-default location, they will be found automatically. If not, you can define the 
-following variables to help `cmake` find them: 
-
-   For All:
-```
-CMAKE_PREFIX_PATH=<path to depended libraries root>
-```
-Note that ```CMAKE_PREFIX_PATH``` may be not shown in the `cmake` GUI. You can use 
-`Add Entry` button to add the variable manually. Type is `PATH`.
-The directory structure should be similar to the following:
-```
-${CMAKE_PREFIX_PATH}/include/pthread.h
-${CMAKE_PREFIX_PATH}/include/mbedtls/... (if mbedtls is used)
-${CMAKE_PREFIX_PATH}/include/openssl/... (if openssl or libressl is not in default location)
-${CMAKE_PREFIX_PATH}/lib/pthreadVC2.lib
-${CMAKE_PREFIX_PATH}/lib/crypto.lib (if openssl or libressl is not in default location)
-${CMAKE_PREFIX_PATH}/lib/mbedcrypto.lib (if mbedtls is used)
-```
-It's better to add the entry before clicking `Configure`, or the installation in
-system will be used instead of the one in `${CMAKE_PREFIX_PATH}`.
-
-For OpenSSL or LibreSSL:
-```
-OPENSSL_ROOT_DIR=<path to OpenSSL installation>
-OPENSSL_LIBRARIES=<path to all the openssl libraries to link>
-OPENSSL_INCLUDE_DIR=<path to the OpenSSL include dir>
-```
-For MbedTLS:
-```
-MBEDTLS_PREFIX=<path to mbedtls installation, default is the same to CMAKE_PREFIX_PATH>
-```
-For pthread:
-```
-PTHREAD_INCLUDE_DIR=<path to where pthread.h lies>
-PTHREAD_LIBRARY=<path to pthread.lib>
-```
-Note that if you use the `cmake` command line to have it configured, please 
-use `/` instead of `\` in the path, or error messages may result.
- 
-
-4. For the sake of cmake generation: When you want to have a 64-bit version,
-remember that cmake by some reason adds /machine:X86 to the linker options.
-There are about four variables ended with `_LINKER_FLAGS` in the `CMakeCache.txt`
-file (also available with Advanced checked in CMake GUI). Remove them, or change
-into /machine:X64.
-
-Also, just after you generated the project for MSVC (if you fail or forget to do
-that before the first compiling, you'll have to delete and regenerate all project
-files) then open Configuration Manager **exactly** after generation from cmake and
-setup x86 platform with requesting to generate this for every subproject.
-
-5. IMPORTANT FOR DEVELOPERS AND CONTRIBUTORS: If you make any changes that fix
-something in the Windows version, remember to keep the project working also for
-all other platforms. To simplify the verification if you just would like to do
-it on the Windows machine, please install Cygwin and make another build for Cygwin,
-for example (remember that 'configure' script requires tcl8.5 package):
-
-		mkdir build-cygwin
-		cd build-cygwin
-		../configure --prefix=install --cygwin-use-posix
-		make
-
-The Cygwin platform isn't any important target platform for this project, but it's
-very useful to check if the project wouldn't be build-broken on Linux.
-
-
+Follow the Windows build intructions: [doc](docs/build-win.md).
```

Comment:
Why not:

Follow the [Windows build instructions](docs/build-win.md).

?

(also there's a typo here)

Reply:
Fixed